apidoc/v2/openapi.yaml

```diff
@@ -1214,14 +1214,54 @@ paths:
                 items:
                   $ref: '#/components/schemas/experiment'
   /experiments/{id}:
-    summary: Actions on an experiment
+    summary: Actions on a specific experiment
     parameters:
       - name: id
         in: path
         description: ID of the experiment
         required: true
         schema:
           type: integer
+    post:
+      tags: ['Experiments']
+      summary: Duplicate an experiment with its ID
+      operationId: post-experiment-by-id
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The unique identifier of the experiment to duplicate.
+          schema:
+            type: integer
+      requestBody:
+        description: Parameters for duplicating an experiment
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                action:
+                  type: string
+                  description: >-
+                    The "duplicate" action creates a new entity based from the given entity id.
+                  default: 'duplicate'
+                copyFiles:
+                  type: boolean
+                  description: >-
```

Comment:
use `|` as elsewhere in the document instead of `>-`

Reply:
thanks, i'm correcting other cases as well then since it was misleading